json/src/main/java/io/airlift/json/ObjectMapperProvider.java

```diff
@@ -215,7 +216,15 @@ public ObjectMapper get()
 
         objectMapper.addModules(modules);
 
-        return objectMapper.build();
+        ObjectMapper mapper = objectMapper.build();
+
+        // Disable the length limit, caller will be responsible for validating the input length
+        mapper.getFactory().setStreamReadConstraints(StreamReadConstraints
```

Comment:
is `mapper.getFactory()` same field as our `jsonFactory` ?

Reply:
No. Internal jackson thingy